### PR TITLE
Feature/tdp 2724 import a preparation

### PR DESCRIPTION
**Pull request by charlesnguyen**
**Link to the JIRA issue**
e.g. https://jira.talendforge.org/browse/TDP-2724

**Please check if the PR fulfills these requirements**
- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [x] The new code does not introduce new technical issues (sonar / eslint)
- [ ] Functional tests have been performed
- [ ] Docker configuration files for config-std or config-cloud profiles are impacted

**Please check the browsers you've tested on**
- [ ] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !

**(Optional) What is the current behavior?**
(Additional information to the Jira)

**(Optional) What is the new behavior?**
(Additional information to the Jira)

**(Optional) Other information**:

### CODE REVIEW

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/196/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/196/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/205/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/205/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/208/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/208/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/211/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/211/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/214/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/214/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/220/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/220/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/228/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/228/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/229/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/229/)

File: dataprep-webapp/src/app/components/widgets-containers/inventory-list/inventory-list-controller.js [hunk 1]
```diff
@@ -188,6 +188,7 @@ export default class InventoryListCtrl {
 			icon: actionSettings.icon,
 			label: actionSettings.name,
 			bsStyle: actionSettings.bsStyle,
+			tooltipLabel: actionSettings.toolTip ? actionSettings.toolTip : actionSettings.name,
```
Comment (ncomont) [inline dataprep-webapp/src/app/components/widgets-containers/inventory-list/inventory-list-controller.js]: Could be `tooltipLabel: actionSettings.toolTip || actionSettings.name,`

File: dataprep-webapp/src/app/components/home/preparation/home-preparation-container.js [hunk 2]
```diff
@@ -28,6 +28,7 @@ const HomePreparationContainer = {
 				view-key="'listview:preparations'"
 				folder-view-key="'listview:folders'"
 			></inventory-list>
+			<div insertion-home-preparation></div>
```
Comment (ncomont) [inline dataprep-webapp/src/app/components/home/preparation/home-preparation-container.js]: Why didn't you use the `<insertion-home-preparation>` element ?

Comment (ncomont) [inline dataprep-webapp/src/app/components/home/preparation/home-preparation-container.js]: And I can't find the directive declaration. Where is it ?

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/230/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/230/)

File: dataprep-webapp/src/app/components/home/preparation/home-preparation-container.js (hunk 2 above)
Comment (charlesnguyen) [inline dataprep-webapp/src/app/components/home/preparation/home-preparation-container.js]: in EE

File: dataprep-webapp/src/app/components/widgets-containers/inventory-list/inventory-list-controller.js (hunk 1 above)
Comment (charlesnguyen) [inline dataprep-webapp/src/app/components/widgets-containers/inventory-list/inventory-list-controller.js]: ok

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/231/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/231/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/232/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/232/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/238/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/238/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/244/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/244/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/245/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/245/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/246/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/246/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/257/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/257/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/270/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/270/)

**Pull request merged**